modules/statistics/server/controllers/statistics.server.controller.js

```diff
@@ -187,6 +188,51 @@ exports.getPushRegistrationCount = function (callback) {
   });
 };
 
+/**
+ * Generate statistics based on the user last seen attribute
+ */
+exports.getLastSeenStatistic = function (type, callback) {
+  var now = moment();
+  var query = { };
+
+  switch (type) {
+    case 'never':
+      query.seen = { $exists: false };
+      break;
+    case 'today':
+      query.seen = { '$gte': now.startOf('day') };
+      break;
+    case 'week':
+      query.seen = { '$gte': now.startOf('week') };
+      break;
+    case 'thisMonth':
+      query.seen = { '$gte': now.startOf('month') };
+      break;
+    case 'sinceAMonthAgo':
+      var thisMonth = now.startOf('month');
+      query.seen = { '$gte': thisMonth.subtract(1, 'months') };
+      break;
+    case 'sinceSixMonthsAgo':
+      var thisMonth = now.startOf('month');
+      query.seen = { '$gte': thisMonth.subtract(2, 'months') };
+      break;
+    case 'sinceAYearAgo':
+      var thisMonth = now.startOf('month');
+      query.seen = { '$gte': thisMonth.subtract(12, 'months') };
+      break;
+    default:
+      return callback(new Error('Missing last seen statistic type.'));
+  }
+  User.count(query, function (err, count) {
+    if (err) {
+      callback(err);
+      return;
+    }
+    callback(null, parseInt(count) || 0);
```

Comment:
A nitpick: could you add `radix` number 10 to each `parseInt(count, 10)` ([You can read here  why](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt#Description)).